### PR TITLE
fx: fix github actions tests failing reporting and coveralls

### LINKED ISSUE #123: fix coveralls reporting and test coverage on github actions

### PR DESCRIPTION
**Pull request by musllim**
# Description

we were getting a green check mark indicating that tests were passing, though they were failing, this fix will stop coveralls reporting and will give a red failing cross on Github actions if tests are failing.

Fixes #123 

# Type of change

Please check the options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have created a loom video for the new feature or enhancement
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by shyakadavis — APPROVED** [review]
LGTM

**Review by emilenfc — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "test": "cross-env NODE_ENV=test && yarn migration && yarn seed:test && yarn jest --coverage --detectOpenHandles",
     "test:dev": "cross-env NODE_ENV=development & yarn migrate:dev && yarn seed:dev && NODE_ENV=development yarn jest --coverage --detectOpenHandles",
     "test:watch": "yarn test -- --watch",
-    "coverage": "jest --coverage --detectOpenHandles --coverageReporters=text-lcov | coveralls",
+    "coverage": "cross-env NODE_ENV=test && yarn migration && yarn seed:test && yarn jest --coverage --detectOpenHandles --coverageReporters=text-lcov | coveralls",
```
Comment (mucyoalexaxel) [inline package.json:13]: @musllim add a way to clear the db once tests have passed. this will enable other PRs to not encounter any migration errors

**Review by mucyoalexaxel — CHANGES_REQUESTED** [review]
@musllim Look into the suggested change.

File: package.json (hunk 1 above)
Comment (musllim) [reply package.json:13]: I have done it in github actions, please check again

Comment (musllim) [reply package.json:13]: I have done it in github actions, please check again

**Pull request merged**